### PR TITLE
Issue #15218: Formatted input file of section 5.3 Camel case: defined to follow google style guide rules

### PR DESCRIPTION
**Pull request by Zopsss**
#15218

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java [hunk 1]
```diff
@@ -43,10 +43,9 @@ void XmlHttpRequest() {}
             void YouTubeImporter() {}
 
             void YoutubeImporter() {}
-    };
-}
+          };
 
-class AbbreviationsIncorrect {
+  class AbbreviationsIncorrect {
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java:48]: Move this extra class inside the main class, reason mentioned here: https://github.com/checkstyle/checkstyle/pull/15239#discussion_r1676801130

**Review by romani — APPROVED** [review]
ok to merge,
this is very easy diff with spaces on same lines, I think I merge it on single review.

File: src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java [hunk 2]
```diff
@@ -59,26 +58,27 @@ void XMLHTTPRequest() {}
...
-        void XMLHTTPRequest() {}
-        // violation above 'XMLHTTPRequest.* more than '1' .* capital letters.'
+      void XMLHTTPRequest() {}
+      // violation above 'XMLHTTPRequest.* more than '1' .* capital letters.'
     }
 
-        InputCamelCaseDefined anonymousBad
-            = new InputCamelCaseDefined() {
+    InputCamelCaseDefined anonymousBad =
+            new InputCamelCaseDefined() {
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java:72]: this is correct update, `=` should be on end of line

Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java:72]: Yeah, Formatter did it

**Pull request merged**